cli/smartcontract/smart_contract.go

```diff
@@ -130,27 +130,27 @@ func NewCommands() []cli.Command {
 				Flags: []cli.Flag{
 					cli.StringFlag{
 						Name:  "in, i",
-						Usage: "Input file for the smart contract to be compiled",
+						Usage: "Input file for the smart contract to be compiled (*.go file or directory). Omit all output-related flags to autocomplete",
 					},
 					cli.StringFlag{
 						Name:  "out, o",
-						Usage: "Output of the compiled contract",
+						Usage: "Output of the compiled contract (omit all output flags for autocompletion)",
 					},
 					cli.BoolFlag{
 						Name:  "verbose, v",
 						Usage: "Print out additional information after a compiling",
 					},
 					cli.StringFlag{
 						Name:  "debug, d",
-						Usage: "Emit debug info in a separate file",
+						Usage: "Emit debug info in a separate file (omit all output flags for autocompletion)",
```

Comment:
But is it enabled by default then? We usually don't produce debugs at all if there is no `-d` and it can be significant for #2428.

Reply:
Excluded from the list of auto-generated files.